### PR TITLE
add docs for toplyne destination action

### PR DESCRIPTION
**Pull request by ankitjena**
### Proposed changes
- Added documentation for Toplyne destination action in segment

### Merge timing
- Depends on the Toplyne destination action PR on `action-destinations` repo.

### Related issues (optional)
https://github.com/segmentio/action-destinations/pull/1068

### CODE REVIEW

File: src/connections/destinations/catalog/toplyne/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+# The end name should be similar to `Slack  Destination`
+title: <destination_name>
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/toplyne/index.md:3]:
```suggestion
title: Toplyne (Actions) Destination
```

File: src/connections/destinations/catalog/toplyne/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+hide-boilerplate: true
+hide-dossier: true
+---
+
+<!-- This template is meant for Actions-based destinations that do not have an existing Classic or non-Actions-based version. For Actions Destinations that are a new version of a classic destination, see the doc-template-update.md template. -->
+
+{% include content/plan-grid.md name="actions" %}
+
+[Toplyne](https://www.toplyne.io/) is a headless Sales AI for revenue teams. Our AI captures and understands every user click stored in your Segment and enriches it with demographic data (through 3rd party enrichment tools). We then, deliver opportunities for expansion and conversion into your existing CRM. We do this without involving your data and engineering teams.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/toplyne/index.md:12]:
```suggestion
[Toplyne](https://www.toplyne.io/) is a headless Sales AI for revenue teams. Toplyne's AI captures and understands every user click passed through Segment and enriches it with demographic data (through 3rd party enrichment tools). Toplyne then delivers opportunities for expansion and conversion into your existing CRM without involving your data and engineering teams.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/toplyne/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure Toplyne**.
+4. Select an existing Source to connect to Toplyne (Actions).
+5. To obtain the API key you have to move to the **Toplyne dashboard**
+6. From your Toplyne dashboard, navigate to **Settings** > **API Settings** and click on **+ Generate new token**.
+7. Enter a name for your token, and select 'Does not expire' under _Expiration_. Then click **Generate new token**
+8. Copy the access token that has just been generated. This will be the API key that you will enter in Segment
+9. Enable the destination and save changes
```
Comment (stayseesong) [inline src/connections/destinations/catalog/toplyne/index.md:41]:
```suggestion
9. Save your changes and enable the destination.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
@ankitjena I'm closing this PR out as I created an editable version for the docs team at #4387  so that we can get this out today

**Pull request closed without merging**